commercial/conf/routes

```diff
@@ -56,3 +56,6 @@ GET         /commercial/ias-passback/:size                                  comm
 
 # AMP iframe HTML, needed for AMP (DCR repo)
 GET         /commercial/amp-iframe.html                                     commercial.controllers.AmpIframeHtmlController.renderAmpIframeHtml()
+
+# DFP Non refreshable line items
+GET         /commercial/dfp-non-refreshable-line-items.json                 commercial.controllers.DfpNonRefreshableLineItemsController.getIds
```

Comment:
Good idea, I think since DFP occurs so much in the Scala source and is also included in the name of the official API it might be better to drop DFP entirely rather than replace it with GAM and having duplicated names for same thing. This should be fine as non-refreshable-line-items should clearly define what the endpoint provides?